Backend/src/server.ts

```diff
@@ -8,19 +8,34 @@ import cors from 'cors';
 import schema from './schema';
 import { ApiError } from './customErrors/apiError';
 import authMiddleware from "./middlewares/basicAuth";
+import initPassport from './middlewares/passportOauth';
+import passport from 'passport';
 
+initPassport();
```

Comment:
Very nice that you put it in a seperate file, so it doesn't clutter server.ts like I had done. 🚀